macro-analytics/Code.js

```diff
@@ -0,0 +1,560 @@
+
+function onOpen() {
+    var ui = SpreadsheetApp.getUi();
+    // Or DocumentApp or FormApp.
+    ui.createMenu('Analytics Helper')
+        .addItem('Setup data', 'setupData')
+        .addItem('Create summary', 'createSummary')
+        .addToUi();
+  };  
+  
+  function setupData() {
+    var spreadsheet = SpreadsheetApp.getActive();
```

Comment:
It looks like this code was generated from a macro. Can you add a comment here explaining that? And adding a link on how to record a macro into a function like this.